pkg/k8s/service_test.go

```diff
@@ -739,25 +739,70 @@ func TestService_Equals(t *testing.T) {
 }
 
 func (s *K8sSuite) TestServiceString(c *check.C) {
-	k8sSvc := &slim_corev1.Service{
-		ObjectMeta: slim_metav1.ObjectMeta{
-			Name:      "foo",
-			Namespace: "bar",
-			Labels: map[string]string{
-				"foo": "bar",
+	tests := []struct {
+		name      string
+		service   *slim_corev1.Service
+		svcString string
+		equals    bool
+	}{
+		{
+			"k8s-ipv4-only-clusterip-service",
+			&slim_corev1.Service{
```

Comment:
Usually we specify the name of the field when making table-driven tests just for the sake of readability